Gemfile

```diff
@@ -30,14 +33,14 @@ group :test do
 end
 
 # Dependencies for the HTTP service
-gem 'sinatra'
-gem 'sinatra-contrib'
-gem 'rack'
-gem 'forwardable'
-gem 'faraday'
-gem 'httpclient'
-gem 'attr_extras'
-gem 'hashie'
+gem 'sinatra', '~> 2.1.0'
```

Comment:
pinning FTW! thanks

Reply:
Something that's important to note here, these pinned versions were simply pulled from the existing _Gemfile.lock_ values. Some of them _are not_ compatible with Enterprise Search. It's not an issue for now because our gemspec has very limited dependencies, but it could become a problem if we ever need to add any of these to the gemspec.